tox.ini

```diff
@@ -1,6 +1,7 @@
 [tox]
 env_list =
     compatibility
+    s3
```

Comment:
I would rename this `mocks3` or something to be more explicit about why it is separated